core/controlplane/config/templates/cluster.yaml

```diff
@@ -836,7 +836,15 @@ worker:
 #     # See https://github.com/kubernetes-incubator/kube-aws/issues/230 for more information.
 #     rbac:
 #       enabled: true
-#
+#   # This flag disables automatic security group rule management for ELBs.  
```

Comment:
I was confused "ELBs" with the one for a k8s api(`controller.loadBalancer`)

Perhaps you'd better express this like "When set to true, configures k8s aws provider so that it doesn't modify every node's security group to include an additional ingress rule per an ELB created for a k8s service whose `type` is `LoadBalancer`"?
Also providing a link to a relevant issue like https://github.com/kubernetes/kubernetes/issues/26670 would help users how to utilize the flag.